go/vt/vtctl/vtctl.go

```diff
@@ -1891,9 +1891,9 @@ func commandReshard(ctx context.Context, wr *wrangler.Wrangler, subFlags *flag.F
 }
 
 func commandMoveTables(ctx context.Context, wr *wrangler.Wrangler, subFlags *flag.FlagSet, args []string) error {
-	workflow := subFlags.String("workflow", "", "Workflow name. Will be used to later migrate traffic.")
+	workflow := subFlags.String("workflow", "", "Workflow name. Can be any descriptive string. Will be used to later migrate traffic via SwitchReads/SwitchWrites.")
 	cells := subFlags.String("cells", "", "Cell(s) or CellAlias(es) (comma-separated) to replicate from.")
-	tabletTypes := subFlags.String("tablet_types", "", "Source tablet types to replicate from.")
+	tabletTypes := subFlags.String("tablet_types", "", "Source tablet types to replicate from (e.g. master, replica, rdonly). Defaults to -vreplication_tablet_type parameter value for the tablet.")
```

Comment:
@rohit-nayak-ps is the default value correct? I thought we default to `REPLICA`.

Reply:
Default is -vreplication_tablet_type (which has REPLICA as the default) - go/vt/vttablet/tabletmanager/vreplication/engine.go:74

Jacques, maybe we should add that here so that it is clear?

Reply:
Added the default for -vreplication_tablet_type in the help